instrumentation-core/spring/spring-webflux-5.0/src/main/java/io/opentelemetry/instrumentation/springwebflux/client/WebClientTracingFilter.java

```diff
@@ -34,14 +35,24 @@
 
 public class WebClientTracingFilter implements ExchangeFilterFunction {
 
+  private final Tracer tracer;
+
+  public WebClientTracingFilter() {
+    this.tracer = TRACER;
+  }
+
+  public WebClientTracingFilter(Tracer tracer) {
+    this.tracer = tracer;
+  }
+
   public static void addFilter(final List<ExchangeFilterFunction> exchangeFilterFunctions) {
```

Comment:
I think add an overload that takes a `Tracer` arg for users who want to provide their own `Tracer`